dev/new-rule/new-rule-template.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const Rule = require('./base');
+const AstNodeInfo = require('../helpers/ast-node-info');
+
+// *** Change template to 'Real Error Message to Report' ***
+const ERROR_MESSAGE = 'Error Message to Report';
+
+// *** Change template to `RealRuleName` ***
+module.exports = class MyNewRule extends Rule {
```

Comment:
would it make sense to put a placeholder here instead of `MyNewRule` and replace it before writing the generated file?

Reply:
ok, did it. 👍